buddyboss/buddyboss-platform
# 777 - GIPHY Enhancement

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #777 

### How to test the changes in this Pull Request:

1. Add GIF file at any place Activity Feed or Message
2. Check at Backend Media, it is getting upload at in it.
3. GIF should direct the show from GIPHY

### Proof Screenshots or Video

https://www.loom.com/share/7407591fe49b49a7a423dc82875185d1

### Other information:




### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.
